frontend/src/App.tsx

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import {BrowserRouter} from "react-router-dom";
+import Root from "./pages/Root";
+
 
 function App() {
+
+
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <div>
+          <BrowserRouter>
+              <Root/>
+          </BrowserRouter>
+      </div>
+
+
   );
 }
 
 export default App;
+
+{
```

Comment:
What are these brackets used for?

Reply:
Remnants of that commented out code, I guess
_Changed_